src/pybind/mgr/dashboard/constraints.txt

```diff
@@ -1,6 +1,6 @@
 CherryPy==13.1.0
 more-itertools==4.1.0
-PyJWT==1.6.4
+PyJWT==2.0.1
```

Comment:
Just curious: did this catch the issue via pylint/unit tests? If Ubuntu Bionic is still using jwt 1.x, we can have a separate 'contraints-ubuntu-bionic.txt' and run pylint/unit-tests in a matrix style.

Reply:
No, this was preventing Ceph API tests job from failing:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/run-backend-api-tests.sh#L49

Reply:
I provided the python unit test that would have caught this issue.